doc/rst/source/pstext.rst

```diff
@@ -51,21 +51,15 @@ Examples
 .. include:: explain_example.rst_
 
 To plot just the red outlines of the (lon lat text strings) stored in the
-file text.txt on a Mercator plot with the given specifications, use
-
-   ::
+file text.txt on a Mercator plot with the given specifications, use::
 
     gmt pstext text.txt -R-30/30/-10/20 -Jm0.1i -P -F+f18p,Helvetica,-=0.5p,red -B5 > plot.ps
 
-To plot a text at the upper left corner of a 10 cm map
-
-   ::
+To plot a text at the upper left corner of a 10 cm map::
 
     echo TopLeft | gmt pstext -R1/10/1/10 -JX10 -F+cTL -P > plot.ps
```

Comment:
```suggestion
    echo TopLeft | gmt pstext -R1/10/1/10 -JX10 -B -F+cTL -P > plot.ps
```

Reply:
Don't this and the next pstext example need ```-Baf``` to avoid warnings if gmt.history doesn't already exist?

Reply:
Oh yes, I forgot to delete the gmt.history file.